docs/kubernetes.rst

```diff
@@ -40,11 +40,44 @@ that has the ability to mutate pod objects before sending them to the Kubernetes
 for scheduling. It receives a single argument as a reference to pod objects, and
 is expected to alter its attributes.
 
-This could be used, for instance, to add sidecar or init containers
+This could be used, for instance, to add init containers
```

Comment:
It can be used for sidecar containers too can't it?

Reply:
How is is made and used for sidecar?  
We can mutation pod object by using pod_mutation_hook, but I think we can not add any containers, excluding the init container ( in the case of using kubernetes_request_factory.py ).

Reply:
@ashb ping?
Could you confirm the document?